From 1e0cdaf23fcd038360b13af831598099a29723af Mon Sep 17 00:00:00 2001
From: shadow_walker <shadowwalkersb@gmail.com>
Date: Sun, 10 Jan 2021 13:27:45 -0600
Subject: [PATCH 1/8] patch for osx arm migration

---
 m4/sasl2.m4 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/m4/sasl2.m4 b/m4/sasl2.m4
index 56e0504a..dd24c242 100644
--- a/m4/sasl2.m4
+++ b/m4/sasl2.m4
@@ -339,7 +339,7 @@ int main(void)
 ],	
 	[ AC_DEFINE(HAVE_GSS_SPNEGO,,[Define if your GSSAPI implementation supports SPNEGO])
 	AC_MSG_RESULT(yes) ],
-	AC_MSG_RESULT(no))
+	AC_MSG_RESULT(no), AC_MSG_RESULT(yes))
   LIBS="$cmu_save_LIBS"
 
 else
